cypress/e2e/gui/home-page-gui.cy.js

```diff
@@ -0,0 +1,45 @@
+/// <reference types="Cypress" />
+
+describe("EngageSphere", () => {
+  beforeEach(() => {
+    cy.visit("https://engage-sphere.vercel.app/");
+  });
+
+  it("Renderiza o cabeçalho com um h1 e alternância de tema", () => {
+    cy.get("h1").contains("EngageSphere");
+    cy.get("#theme-toggle-button").should("be.visible").click();
+    cy.get("h1").contains("EngageSphere");
+  });
```

Comment:
Deixe uma linha em branco entre cada caso de teste para melhorar a legibilidade do código.

```suggestion
  });
  
```